gtk/tests_backend/window.py

```diff
@@ -15,6 +15,8 @@ class WindowProbe(BaseProbe):
     supports_multiple_select_folder = True
     supports_move_while_hidden = False
     supports_unminimize = False
+    supports_minimize = not BaseProbe.IS_WAYLAND
+    supports_placement = not BaseProbe.IS_WAYLAND
```

Comment:
As best I can gather, Wayland just doesn't support allowing individual applications take command of aspects of the broader windowing environment like X does.

Reply:
Yeah - I've seen variations on that discussion: fundamental philosophical arguments about where the app's responsibility ends and the window manager starts.